Gemfile.lock

```diff
@@ -245,33 +247,34 @@ GEM
     method_source (1.0.0)
     mime-types (3.3.1)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2019.1009)
-    mimemagic (0.3.4)
+    mime-types-data (3.2020.0512)
```

Comment:
### [mime-types-data](https://github.com/mime-types/mime-types-data/)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.2019.1009 → 3.2020.0512

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **15** commits. See the full changes on [the compare page](https://github.com/mime-types/mime-types-data/compare/v3.2019.1009...v3.2020.0512).

These are the first 10 commits:
- (bc8466f) [Add entry for application/x-ms-dos-executable](https://github.com/mime-types/mime-types-data/commit/bc8466f9c776b0063513115884e592664a9855e7)
- (ac8a605) [Remove .exe extension from application/octet-stream MIME typ…](https://github.com/mime-types/mime-types-data/commit/ac8a60594d0470ef9738160357e2dd1a4b09e03b)
- (379fb39) [Add entry for audio/x-m4a](https://github.com/mime-types/mime-types-data/commit/379fb3984c256087584ef73085672248ef59c41a)
- (367aee2) [Add application/x-smarttech-notebook](https://github.com/mime-types/mime-types-data/commit/367aee27d3b2cd539a797b9f09a9ea97c166d3c9)
- (f7dbf06) [Add audio/wav](https://github.com/mime-types/mime-types-data/commit/f7dbf06876f2d9f0cc0d0f1a19ff3c60454be808)
- (3f88205) [Add "raw" image mime types](https://github.com/mime-types/mime-types-data/commit/3f88205cb5bfcc46e7b9d97d9a015029488bd174)
- (894e24c) [Add "raw" image mime types (#33)](https://github.com/mime-types/mime-types-data/commit/894e24c8293af15d1cfffa29206475400c5e40c9)
- (33a48a1) [Merge pull request #31 from sutori/audio-wav](https://github.com/mime-types/mime-types-data/commit/33a48a12407d18aac113f88dbea264a688b123f2)
- (b4ed4a8) [Merge pull request #30 from sutori/x-smarttech-notebook](https://github.com/mime-types/mime-types-data/commit/b4ed4a8a1242a4f30d3ef67c11430d0b85a33c07)
- (1650bd2) [Merge pull request #29 from sutori/x-m4a](https://github.com/mime-types/mime-types-data/commit/1650bd2dc4f4de1611425d80b4effaaf763ba0a0)


</details>